client/my-sites/plans/jetpack-plans/product-store/items-list/style-most-popular.scss

```diff
@@ -22,13 +22,13 @@
 	list-style-type: none;
 	display: grid;
 	grid-template-columns: 1fr;
-	grid-gap: 24px;
+	grid-gap: 30px;
```

Comment:
Is this gap related to the change in design? It seems counter-intuitive to add more columns in the row and increase the gap size between them. But it also makes sense if it makes the design more esthetic.

Reply:
Yep, it's related to the design change proposed. It looks counterintuitive, but the hero image is now at the top of the card instead of on the side - that's why the gap size can be bigger.